### ISSUE TITLE
[Suggestions for upcoming new dissension]

### ISSUE THREAD

**Issue by tyl3r99** [genesis92x/Dissension#3]
**Hewwoo!!
Im gonna post some ideas I have for the new dissension and I will do my best to post them in categories and explain them in detail.** 

### [Server configuration/Parameter settings]
_As we all know dissension auto detects the mods available and are able to choose what faction we want to be included in the playthrough._ 
> - Ability to set a custom faction through an array of classnames for an example this is from the liberation preset config files.
[](https://prnt.sc/usu0db
https://prnt.sc/usu0h1
https://prnt.sc/usu0ts)
> - Similar to the post above, but the ability to customise the arsenal available to players. (Some groups may not want people running about in csat gear with viper helmets and a pink backpack. Liberation has combatted this by having set arsenal presets as shown in the link below. 
https://prnt.sc/usuaqu
https://prnt.sc/usublx
You are able to blacklist items but apparently it is more performance friendly having a whitelist option. If that makes sense.
As shown above, you are able to fully customise what units you want involved and how much the units are to purchase.
> - Ability to choose what personality each commander which includes a random option which chooses automatically. 
> - Include a no stamina and weapon sway parameter option. (self explanatory).
> - Include a wipe save game parameter, which deletes the data saved that relates to dissension, in the serverprofile.vars so we can manually start a new game. 
> - Virtual unit movement speed parameter. (so we can select how fast virtualised units move when cached). 
> - Time acceleration parameter setting (sets how fast the time of day passes during a game). 
> - feature to disable the custom medical system for those who play with ACE. (as it could conflict with the ace medical system.
> - Make it so the parameters selected at the start of the playthrough are saved until the game is either completed or manually wiped. _(like in liberation - they have a parameter that loads the progress by default and the user for the start of the game selects the SAVE option, but at each restart the parameter reverts back to load.)_ _- I can go into this more in-depth but ideally be better to show you via screen share what I mean._ 
> - Civilian presence (Very low, low, medium, high).
> - Enable/Disable VCOM (this suggestion is purely for those groups who don't use vcom and wish to use their modpack which might... contain ASR_AI for example. Which caters for more of an audience).
> - Ability to set what distance virtual units become spawned in when a player gets x distance away.
### [Side mission ideas]
> - Hostage rescue. (gives you intel of where the enemy are pushing for a limited amount of time?)
> - HVT kill. (could de moralise the enemy commander?)
> - FOB hunt (spawns a composition of a fob and the players have to destroy specific item within the fob to complete it and maybe get rewards for doing so). (resource award)
> - Convoy ambush. (resource award)
> - Defend style side mission (Either a town or sector is under attack and your assigned to defend the area.)
> - Destroy X mission (Like destroy a radio tower with satchel charges). 

### [General gameplay ideas]
> - I noticed in the original dissension that when commanders attack towns, they attack in force from 1 direction. 
Any chance of making the commander deploy troops maybe 1 to 3 places and attack in a strategy form? 
> - A player menu or additional menu on the tablet where you can select view distance for like ground, air, vehicles. 
Finally a FPS target feature? where a player can set a figure in a box and the game will automatically adjust to keep at that fps target? (not sure if that's just too much or not) 
> - If not using ace and revive is on = ability for a nearby AI to heal you and wake you up. 
> -

**Comment by tyl3r99** [comment]
edited comment with few changes, additions

**Comment by dvdbrewster** [comment]
I would like to see difficulties (accuracy, general danger, )for the vcom ai’s  (based on their role
, not necessarily Faction -side)  available to change  from like extreme/hard/ medium/ easy either in CBA  or mission ParamS include some ChickenShxx  check box, so u have to go after them.

**Comment by tyl3r99** [comment]
> I would like to see difficulties (accuracy, general danger, )for the vcom ai’s (based on their role
> , not necessarily Faction -side) available to change from like extreme/hard/ medium/ easy either in CBA or mission ParamS include some ChickenShxx check box, so u have to go after them.

Vcom 3.4 has this feature already implemented. If you look in cba settings it has side specific skill settings

**Comment by dvdbrewster** [comment]
> > I would like to see difficulties (accuracy, general danger, )for the vcom ai’s (based on their role
> > , not necessarily Faction -side) available to change from like extreme/hard/ medium/ easy either in CBA or mission ParamS include some ChickenShxx check box, so u have to go after them.
> 
> Vcom 3.4 has this feature already implemented. If you look in cba settings it has side specific skill settings

Yes But, that’s an old version of vcom, and u have to reall know what u r doing..   extreme/ hard / easy ..  would be cooler.

**Comment by tyl3r99** [comment]
Vcom 3.4.0 is the current version of vcom.

3.3.3 was the previous version
Its not hard at all

**Comment by dvdbrewster** [comment]
> Vcom 3.4.0 is the current version of vcom. 3.3.3 was the previous version Its not hard at all
> […](#)

Oh cool.  Default on server..  inside vcom there was medium , hard, easy.. default was usually hard.. but skill parameters .. for hard not always 
very difficult.. and very aggressive..  I sometimes set ‘can flea’  and grade the spec ops up some. 
    I just think ,lol, some  sunlight here for difficulty.  
Like snipers are awesome..  Rey soldiers have accuracy but lack some courage.. etc.  Like custom.      Dissension is Great

**Comment by tyl3r99** [comment]
I mean he could make 2 parameters that adjust the base skill of each side
im not sure. We shall seeee
